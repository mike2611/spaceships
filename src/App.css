body {
  margin: 0;
}

.mission-container {
  width: 90%;
}

.mission-container h2,
.mission-name {
  font-size: 1.4rem;
  font-weight: 500;
}

.member-badge {
  border-radius: 10px;
  width: 11rem;
  color: white;
  font-weight: bold;
  font-size: 1.2rem;
  border: none;
}

.not-member {
  background-color: #6d757d;
}

.member {
  background-color: #18a2b8;
}

.join-btn {
  background-color: transparent;
  border-radius: 5px;
  height: 3rem;
  width: 10rem;
  font-size: 1.4rem;
}

.not-join {
  border: 1px solid #343a40;
  color: #343a40;
}

.join {
  border: 1px solid #dc3645;
  color: #dc3645;
}

.logo-nav {
  margin-right: 20%;
}

.img-logo {
  width: 8%;
}

.title-nav {
  font-size: 2.5rem;
  font-weight: bold;
}

.links-nav {
  font-size: 1.2rem;
  color: #3ca2b8;
}
