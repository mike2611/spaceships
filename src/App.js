import './App.css';
import { Provider } from 'react-redux';
import configureStore from './redux/configureStore';
import Routing from './components/Routing';

function App() {
  const store = configureStore;

  return (
    <Provider store={store}>
      <div>
        <Routing />
      </div>
    </Provider>
  );
}

export default App;
