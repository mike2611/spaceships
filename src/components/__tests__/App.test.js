import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import App from '../../App';

describe('Home component', () => {
  test('should render Home component correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<App />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
