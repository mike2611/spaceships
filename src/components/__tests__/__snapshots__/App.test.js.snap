// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render Home component correctly 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <div>
    <Routing />
  </div>
</Provider>
`;
